ml-proto/spec/f32.ml

```diff
@@ -1,6 +1,7 @@
 include Float.Make(struct
                      include Int32
                      let default_nan = 0x7fc00000l
+                     let alternate_nan = 0xffc00000l
```

Comment:
Nit: Is there a reason to favour one over the other? If not, why not simply call them `pos_nan` and `neg_nan`?